indico/MaKaC/webinterface/rh/conferenceDisplay.py

```diff
@@ -72,6 +73,12 @@ def _process(self):
 
 class RHConferenceBaseDisplay( RHConferenceBase, RHDisplayBaseProtected ):
 
+    def _forbidden_if_not_admin(self):
+        if session.user and session.user.is_admin:
+            flash(_('This page is currently not visible by non-admin users (menu entry disabled)!'), 'warning')
```

Comment:
`if not request.is_xhr` (the RH to generate a static link also inherits from `RHDisplayProtectionBase` and flashing there is somewhat ugly)